app/controllers/pipeline_outputs_controller.rb

```diff
@@ -28,7 +28,13 @@ def show
   end
 
   def show_taxid_fasta
-    @taxid_fasta = get_taxid_fasta(@pipeline_output, params[:taxid])
+    if params[:hit_type] == "NT_or_NR"
+      nt_array = get_taxid_fasta(@pipeline_output, params[:taxid], params[:tax_level].to_i, 'NT').split("^>")
+      nr_array = get_taxid_fasta(@pipeline_output, params[:taxid], params[:tax_level].to_i, 'NR').split("^>")
+      @taxid_fasta = ((nt_array | nr_array) - ['']).join(">")
```

Comment:
cool. but since you split by '>', the first read of each nt_ nr_ array might lose '>'

Reply:
fixed it!